src/scss/custom/_tables.scss

```diff
@@ -1,13 +1,3 @@
-// Site-wide Vanilla table overrides
-table {
-  min-width: 900px;
```

Comment:
This is what made the tables scroll horizontally so I've removed it for all tables.